CSharpSeleniumWebdriverSample/README.md

```diff
@@ -0,0 +1,63 @@
+# CSharpSeleniumWebdriverSample
+
+This sample demonstrates how you might set up a CI build for a simple, static html page to perform end to end accessibility tests in a browser.
+
+## Getting Started
+
+The individual files in the sample contain comments that explain the important parts of each file in context.
+
+Some good places to start reading are:
+
+* [tests/SampleTest.cs](./tests/SampleTest.cs): C# test file that opens [src/samplePage.html](./src/samplePage.html) in a browser with Selenium and runs accessibility scans against it with Selenium.WebDriver.ChromeDriver and Selenium.WebDriver.GeckoDriver
+* [azure-pipelines.yml](./azure-pipelines.yml): Azure Pipelines config file that sets up our Continuous Integration and Pull Request builds
+
+## Tools and libraries used
+
+* [Microsoft.NET.Test.Sdk](https://github.com/microsoft/vstest/) to enable running tests, collect diagnostics data and report results.
+* [Microsoft.NETCore.App](https://dotnet.microsoft.com) a set of .NET API's that are included in the default .NET Core application model.
+* [MSTest.TestAdapter and MSTest.TestFramework](https://github.com/microsoft/testfx) MSTest V2 adapter to discover and execute MSTest Framework based tests.
+* [Selenium.Axe](https://github.com/TroyWalshProf/SeleniumAxeDotnet) to run an axe accessibility scan on the page from the Selenium browser.
+* [Selenium.WebDriver.ChromeDriver](https://github.com/jsakamoto/nupkg-selenium-webdriver-chromedriver/) to enable Selenium to drive Chrome.
```

Comment:
We should probably also include "Selenium.WebDriver" itself here (maybe adding an explicit PackageReference for it in the csproj too)

Reply:
We don't use it in the project, do we still need it?

Reply:
We do use it in the project (ctrl+F for `using OpenQA.Selenium;`) - we just happen to be getting it indirectly via the `Selenium.Axe` dependency. But I think it's an important enough piece of the sample that we should note it explicitly in this list